lib/purescript-cst/src/Language/PureScript/CST/Parser.y

```diff
@@ -147,7 +147,7 @@ many(a) :: { NE.NonEmpty a }
   : many1(a) { NE.reverse $1 }
 
 many1(a) :: { NE.NonEmpty a }
-  : a { pure $1 }
+  : a %shift { pure $1 }
```

Comment:
I think some of these are unnecessary, unless you have a master plan I'm not seeing.

```suggestion
  : a { pure $1 }
```

Reply:
Nah. I just wasn't able to figure out that the `%shift` should go on the `many()` one rather than here.